test/active_model_serializers/register_jsonapi_renderer_test_isolated.rb

```diff
@@ -58,21 +63,6 @@ def test_jsonapi_parser_not_registered
       assert_nil parsers[Mime[:jsonapi]]
     end
 
-    def test_jsonapi_renderer_not_registered
-      expected = {
-        'data' => {
-          'attributes' => {
-            'name' => 'Johnny Rico'
-          },
-          'type' => 'users'
-        }
-      }
-      payload = '{"data": {"attributes": {"name": "Johnny Rico"}, "type": "authors"}}'
-      headers = { 'CONTENT_TYPE' => 'application/vnd.api+json' }
-      post '/render_with_jsonapi_renderer', params: payload, headers: headers
-      assert expected, response.body
```

Comment:
so, this should have raised an exception when hitting `render_with_jsonapi_renderer` which calls` render jsonapi: json` but there's no `jsonapi` renderer registered. good test to have